.github/workflows/ci.yaml

```diff
@@ -34,24 +34,15 @@ jobs:
       with:
         provider: microk8s
         channel: 1.21/stable
+        charmcraft-channel: latest/candidate
 
-    - name: Install dependencies
-      run: |
-        set -eux
-        sudo snap install juju-wait --classic
-        # https://github.com/charmed-kubernetes/actions-operator/issues/27
-        sudo snap refresh charmcraft --channel latest/candidate
-
-    - name: Build and deploy charm
+    # TODO: Remove once the actions-operator does this automatically
+    - name: Configure kubectl
```

Comment:
This might actually be done in action-operator.  I could be wrong but I think it was addressed

Reply:
You were right, [it is done in the [actions-operator](https://github.com/charmed-kubernetes/actions-operator/blob/6febc7aa28a75096c485621f1b269d4732291c29/src/bootstrap/index.ts#L188) now